src/index.jsx

```diff
@@ -1,6 +1,40 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import App from './components/App/App';
+import Header from './components/Header/Header';
+import MovieDetails from './components/MovieDetails/MovieDetails';
+import SearchBar from './components/SearchBar/SearchBar';
+import InfoBar from './components/InfoBar/InfoBar';
+import SortBy from './components/SortBy/SortBy';
+import Main from './components/Main/Main';
+import NotFound from './components/NotFound/NotFound';
 import './index.scss';
+import MovieList from './components/MovieList/MovieList';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <Router>
+    <App>
+      <Switch>
+        <Route path="/">
+          <div>
+            <Header>
+              <Switch>
+                <Route path="/" component={SearchBar} exact />
+                <Route path="/search" component={SearchBar} />
```

Comment:
You can remove `/search` route

Reply:
No, I can't cuz when I press Search button I end up on route `/search?...` and on this route `/search` I have to have `search bar`